tool/grind.dart

```diff
@@ -54,7 +54,7 @@ serveLocalAppEngine() async {
     _serverUrlOption: 'http://127.0.0.1:8084/',
   }),
 ))
-serveLocalBackend() async {
+Future serveLocalBackend() async {
```

Comment:
Is there a lint rule for [avoiding incomplete generic types](https://dart.dev/guides/language/effective-dart/design#avoid-writing-incomplete-generic-types)?

Reply:
Possibly? Or perhaps this would be caught by enabling one of:

```
analyzer:
  language:
    strict-inference: true
    strict-raw-types: true
```